### PR TITLE
Cloud lock integration

### PR DESCRIPTION
**Pull request by Spivaka**
This is not tested yet as we have no access. includes only examples from the mock

### CODE REVIEW

File: Integrations/integration-CloudLocki.yml [hunk 1]
```diff
@@ -0,0 +1,262 @@
+commonfields:
+  id: CloudLock
+  version: -1
+name: CloudLock
+display: CloudLock
+category: Authentication
```
Comment (Spivaka) [inline Integrations/integration-CloudLocki.yml]: Forgot...

File: Integrations/integration-CloudLocki.yml [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
...+wOt6vyfM3k20g8ySgkze352u5w1yPLT5t8O/hCq9RNcVSRp6+WxL2BP5mHJEljNlg3AkmSxsqiYmHxGSyGkyRppAZRGsKMG0biFcYgSWMj601NnpqwGE6SRizvTUy+29UdpI4LblPU2TeAZYYhSdJ4sUOcJEnjY1FRzf8ccGejkCRpxDb0Ox9BsbgkkZ0Uid09ICTNGrvD7Yosn5h8V8BjjELS3J+v7Q63Db1i4+SHI/EPRiEJWF2U+VdL4mjgIuOQJGk82CFOkqTxsKCo5n8Ki+EkSdL4uFckHmYMkjQWUr5i3ZsTHGcUkjRSea+/5D1YDCfp6g4psnQmsK9RSJI0HiyIkyRp9OYXU5v+G7i7UUiSNHqDy5wqA/MiZW8xBkmzyu5wOyvlp617S4p4ilFIGs752u5w13k+npp8Z8DDjULStVhV5IPPgV3WJUkaBy6ZKknSaPWLNP1RgnsYhSRJY2KJU+WC6acBN/FgkDSrPnq2GVyXS+dDVtMvSupUkCIIaiAB8RpIFsNJ0jCuU5fOZ/6CAVWVqFOQBURkkALgddipU9K2JG5ZMP25Eu4JbDAQSZJGx9feJUkanV6xsfdRgvsbhSRJGiO7Ay8yBkkaC6+C9HxjkKRxOB/zXGOQtB3uUGTpk8B8o5AkaXR87V2SpNHIev0l7wni741CkqQxE1WnN79I9QuBvTwQJM2qPRaYwY77V+CFxiBpqFwu9dq8xPOxpB109yJNf7SEY4CBcUiSNHwWxEmSNFtWX8evbzyXeXWi7lXUdUZGEKl+TUR6uKFJkjReBnXe9QgOgHimR4IkjdxxkF5mDJI0cs8mxSuMQdIOC+7f6y153wAeCVQGIknScFkQJ0nS8D2DSM8zBkmSxlDa0unNz8lfDMk2TpJml93hdtQ9gzgFSEYhaajsDndNDwvS641B0s6KxMOKjWt/l7LsWXVdk+WJQZVDymH9CgOSJGkOZUYgSdJQPSCINxiDJEkaQzdMpMcagySN1M2I+iNAzygkaaRum4JTsThZ0q5K6ZmAndglSRoyO8RJkjQ8t0tUH5p5/UuSJI2l6fmd3fSiP3guMN+DQNKssjvcjti/TtlnCXYzCklDZ3e4qzow4NOQFhqFpNkQEW8A1gP/bRqSJA2HHeIkSRqOg6L2RpokSeNssKXT74ztBul4jwJJGt15uCjqzxLsbxSSNNrzc
```
Comment (Spivaka) [inline Integrations/integration-CloudLocki.yml]: this shows on line 7, which line do you mean?

Comment (anara123) [inline Integrations/integration-CloudLocki.yml]: image

**Comment by Spivaka** [comment]
@anara123 I've addressed your comments, please merge and delete

**Comment by Spivaka** [comment]
@anara123

**Pull request merged**